java/core/src/test/java/com/microsoft/bond/testmodel/schema.bond

```diff
@@ -7,6 +7,8 @@ struct C;
 struct D;
 struct E<U>;
 
+struct NotHere;
```

Comment:
FYI, when you rebase your changes on master, you'll pick up the gradle-native codegen. This file will want to live somewhere under `java/core/src/test/bond`.